Remotion/Web/Core/WebStringConverter.cs

```diff
@@ -100,7 +100,7 @@ private object ConvertToString (in WebString input)
 
       if (input.Type == WebStringType.PlainText)
       {
-        if (rawString.StartsWith (c_htmlPrefix))
+        if (rawString.StartsWith (c_htmlPrefix) || rawString.StartsWith (c_textPrefix))
```

Comment:
Why this change?

Reply:
Without it the conversion is not correct in the special case that the text starts with `(text)`. For example:
 - we have a WebString with text `(text)asd` and a type plaintext
 - we convert it to a `string` using `ConvertTo` which as of now will return `(text)asd` instead of `(text)(text)asd`
 - we convert it back to a `WebString` using `ConvertFrom` but this will remove the `(text)` prefix leaving us with a web string with text `asd` type plaintext